.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  web:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+    command: sleep infinity
+    ports:
+      - "3000:3000"
```

Comment:
Should it be port 5000?
https://github.com/DFE-Digital/register-trainee-teachers/blob/c057344eefa488c3b00eca84bed19be3487da2b1/config/settings.yml#L1